src/EnvRegistry.ts

```diff
@@ -0,0 +1,60 @@
+import { from } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
+import { messages } from '@capsulajs/capsulajs-configuration-service/lib/utils';
+import { ConfigurationService, ConfigurationServiceLocalStorage } from '@capsulajs/capsulajs-configuration-service';
+import {
+  EnvRegistry as EnvRegistryInterface,
+  EnvRegistryItem,
+  EnvironmentsRequest,
+  EnvironmentsResponse,
+  RegisterResponse,
+} from './api/EnvRegistry';
+import { envKeyValidator, envValidator, validationMessages } from './utils';
+import { ConfigEntry, EntriesResponse } from './types';
+
+export default class EnvRegistry implements EnvRegistryInterface {
```

Comment:
Documentation of the service and the methods is missing

Reply:
Documentation is in the `src/api/EnvRegistry.ts`